contents/javascript/Javascript-framework-React.md

```diff
@@ -0,0 +1,115 @@
+<frontmatter>
+  title: ReactJs
```

Comment:
Should this be React? If not, match with the page heading below.

Reply:
Yep, I missed that. I'll change it.